woocommerce/woocommerce
# Change the Woo logos in the mobile banner and modal in admin

### Changes proposed in this Pull Request:

This pull request is a follow-up of https://github.com/woocommerce/woocommerce/pull/53308, it changes the Woo logo in two places:

1. The "Download the WooCommerce App" banner that appears in the WooCommerce dashboard in Android devices:

![image](https://github.com/user-attachments/assets/295a66dd-fa97-4331-9dd4-b8f931bdf8d9)

2. The "Manage orders and track sales in real-time with the free mobile app" modal window that appears when clicking "Get the free WooCommerce mobile app" link in the "What to do next" box in the WooCommerce dashboard (the mobile devices image and its background pattern are changed too):

![image](https://github.com/user-attachments/assets/137423e7-8464-4642-aea9-dadb0fdc7a0a)

![image](https://github.com/user-attachments/assets/3be35e29-24d0-4205-b020-e03f63e8e348)


### How to test the changes in this Pull Request:

1. Activate the new branding as instructed in https://github.com/woocommerce/woocommerce/pull/53308

2. Open the WooCommerce dahsboard (WooCommerce - Home) from an Android device. Alternatively, you can simulate such a device using the developer tools of your browser, here's how in Chrome:

* Open the developer tools (F12 key)
* Click the "Toggle device toolbar" icon:

![image](https://github.com/user-attachments/assets/ae0c4ebf-59a0-42e8-9859-207c5a9c986d)

* Click on the device selector and select "Edit"
* Create a new device with the word "Android" in the user agent string

3. Verify that the logo next to the "Download the WooCommerce App" text is the new one.

Note: if you have seen and dismissed this banner in the past it won't appear. To restore the logo you can run this in the command line (replace the user id accordingly):

```
wp db query "delete from wp_usermeta where meta_key='woocommerce_admin_android_app_banner_dismissed' and user_id=<user id>"
```

4. Go back to normal display (click the "Toggle device toolbar" icon again) and click the "Get the free WooCommerce mobile app" link in the "What to do next" box. Verify that you see the new logo in the top left corner of the modal window that opens.

5. Deactivate the new branding, repeat the steps, and verify that you see the old logos again.

### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
